.travis.yml

```diff
@@ -4,6 +4,7 @@ os:
   - linux
 julia:
   - 0.6
+  - 0.7
```

Comment:
I was going to say do this to AppVeyor but it looks like it was never setup. Oh well, something for the future

Reply:
I intentionally only tested on one OS